autolink/postlink/activityLinker.js

```diff
@@ -27,24 +56,32 @@ class ActivityLinker {
       debugn("   Removing getMainComponentName function");
       return contents.replace(/\/\*\*\s*\n([^\*]|(\*(?!\/)))*\*\/\s*@Override\s*protected\s*String\s*getMainComponentName\s*\(\)\s*{\s*return.+\s*\}/, "");
     }
+    warnn("   getMainComponentName function was not found.");
```

Comment:
For some cases, the script was running silently and failing/not doing anything without proper feedback. I added some messages for this as wel..